### PR TITLE
Update Collaboration

### PR DESCRIPTION
**Pull request by marek-vysoky**
Adding comments

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @marek-vysoky has signed the CLA. Thanks for the pull request!

File: Box.V2/Managers/BoxCollaborationsManager.cs [hunk 1]
```diff
@@ -51,11 +51,16 @@ public BoxCollaborationsManager(IBoxConfig config, IBoxService service, IBoxConv
         }
 
         /// <summary>
-        /// Used to edit an existing collaboration. Descriptions of the various roles can be found 
-        /// <see cref="https://support.box.com/entries/20366031-what-are-the-different-collaboration-permissions-and-what-access-do-they-provide"/>
+        ///Used to edit an existing collaboration. Descriptions of the various roles can be found here
```
Comment (marek-vysoky) [inline Box.V2/Managers/BoxCollaborationsManager.cs]: Comment is copied from https://docs.box.com/reference#edit-a-collaboration

File: Box.V2/Managers/BoxCollaborationsManager.cs [hunk 2]
```diff
@@ -51,11 +51,16 @@ public BoxCollaborationsManager(IBoxConfig config, IBoxService service, IBoxConv
...
         /// </summary>
-        /// <param name="collaborationRequest"></param>
-        /// <returns></returns>
+        /// <param name="collaborationRequest">
+        /// collaborationRequest.Id (Required) - Id of the collaboration object
+        /// collaborationRequest.Role (Required) - The access level of this collaboration
+        /// collaborationRequest.Status - Whether this collaboration has been accepted
+        /// </param>
```
Comment (marek-vysoky) [inline Box.V2/Managers/BoxCollaborationsManager.cs]: Comments for field is from https://docs.box.com/reference#edit-a-collaboration

File: Box.V2/Managers/BoxCollaborationsManager.cs (hunk 1 above)
Comment (coolboy) [inline Box.V2/Managers/BoxCollaborationsManager.cs]: I mean there should be a space between /// Used, but not ///Used

File: Box.V2/Managers/BoxCollaborationsManager.cs (hunk 2 above)
Comment (coolboy) [inline Box.V2/Managers/BoxCollaborationsManager.cs]: I think you should put a comment for fields from https://docs.box.com/reference#fields

**Comment by marek-vysoky** [comment]
I've added comment for fields and formated ///Used

**Pull request merged**